CODE_OF_CONDUCT.md

```diff
@@ -16,6 +16,6 @@ Attendees violating these rules will be asked to leave the event at the sole dis
 Thank you for helping to make this a welcoming, friendly space for all.
 
 ## Contact Information 📧
-If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact the organizer, Juan Pablo Flores at [education@github.com](mailto:education@github.com) or another GitHub employee.
+If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact the organizer, Juan Pablo Flores via [https://support.github.com/](https://support.github.com/) or any another GitHub employee.

```

Comment:
If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact support or another GitHub employee.